javaslang-benchmark/src/test/java/javaslang/collection/ArrayBenchmark.java

```diff
@@ -1,24 +1,42 @@
-package javaslang.benchmark.collection;
+package javaslang.collection;
 
-import javaslang.benchmark.JmhRunner;
+import javaslang.JmhRunner;
+import org.junit.Test;
 import org.openjdk.jmh.annotations.*;
+import org.openjdk.jmh.infra.Blackhole;
 
 import java.util.*;
 
-import static javaslang.benchmark.JmhRunner.*;
+import static javaslang.JmhRunner.*;
 
 public class ArrayBenchmark {
+    static final Array<Class<?>> CLASSES = Array.of(
+            Create.class,
+            Head.class,
+            Tail.class,
+            Get.class,
+            Update.class,
+            Prepend.class,
+            Append.class,
+            Iterate.class
+    );
+
+    @Test
+    public void testAsserts() {
+        JmhRunner.runDebug(CLASSES);
+    }
+
     public static void main(String... args) {
-        JmhRunner.runQuick(ArrayBenchmark.class);
+        JmhRunner.runNormal(CLASSES);
     }
 
     @State(Scope.Benchmark)
     public static class Base {
-        @Param({ "10", "100", "1000", "10000" })
+        @Param({ "10", "100", "1000" })
         public int CONTAINER_SIZE;
 
-        int expectedAggregate = 0;
-        public Integer[] ELEMENTS;
+        int EXPECTED_AGGREGATE;
```

Comment:
As stated before, personally I would use camel case names here, as the convention is upper case for constants only.